modules/cluster-logging-rn-5.5.6.adoc

```diff
@@ -0,0 +1,49 @@
+//module included in cluster-logging-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-5-6_{context}"]
+= Logging 5.5.6
+This release includes link:https://access.redhat.com/errata/RHBA-2023:0386[OpenShift Logging Bug Fix Release 5.5.6].
+
+[id="openshift-logging-5-5-6-bug-fixes"]
+== Bug fixes
+* Before this update, the Pod Security admission controller added the label `podSecurityLabelSync = true` to the `openshift-logging` namespace. This resulted in our specified security labels being overwritten, and as a result Collector pods would not start. With this update, the label `podSecurityLabelSync = false` preserves security labels. Collector pods deploy as expected. (link:https://issues.redhat.com/browse/LOG-3340[LOG-3340])
+
+* Before this update the Operator would install the console view plugin even when not enabled on the cluster, causing the Operator to crash. With this update, accounts for clusters that do not have the console enabled no longer crash the Operator. (link:https://issues.redhat.com/browse/LOG-3407[LOG-3407])
```

Comment:
Can the second sentence be improved? It seems to imply that "Accounts no longer crash the Operator."

Reply:
Maybe
```suggestion
* Before this update, the Operator installed the console view plugin, even when it was not enabled on the cluster. This caused the Operator to crash. With this update, if an account for a cluster does not have the console view enabled, the Operator functions normally and does not install the console view. (link:https://issues.redhat.com/browse/LOG-3407[LOG-3407])
```